ipam/ippool_manager_test.go

```diff
@@ -178,7 +184,10 @@ var _ = Describe("IPPool manager", func() {
 			}
 			Expect(addressMap).To(Equal(tc.expectedAddresses))
 			Expect(tc.ipPool.Status.Allocations).To(Equal(tc.expectedAllocations))
-			Expect(tc.ipPool.Status.LastUpdated.IsZero()).To(BeFalse())
+			if !reflect.DeepEqual(previousAllocations, tc.ipPool.Status.Allocations) {
+				Expect(tc.ipPool.Status.LastUpdated.IsZero()).To(BeFalse())
+			}
```

Comment:
The opposite should probably be tested too : 

```suggestion
			if !reflect.DeepEqual(previousAllocations, tc.ipPool.Status.Allocations) {
				Expect(tc.ipPool.Status.LastUpdated.IsZero()).To(BeFalse())
			} else {
				Expect(tc.ipPool.Status.LastUpdated.IsZero()).To(BeTrue())
			}
			
```